src/dunst.c

```diff
@@ -78,6 +78,7 @@ static gboolean run(void *data)
                 draw();
                 output->win_show(win);
         } else {
+                LOG_I("Calling win_hide");
```

Comment:
This is redundant (and the information level is too high for this log)

Reply:
I removed it